test/integration/cloudflareCache.test.js

```diff
@@ -0,0 +1,200 @@
+const chai = require("chai");
+const { expect } = chai;
+const chaiHttp = require("chai-http");
+const sinon = require("sinon");
+
+const app = require("../../server");
+const addUser = require("../utils/addUser");
+const authService = require("../../services/authService");
+const cleanDb = require("../utils/cleanDb");
+const logsQuery = require("../../models/logs");
+
+const config = require("config");
+const cookieName = config.get("userToken.cookieName");
+const cloudflare = require("../../services/cloudflareService");
+
+const userData = require("../fixtures/user/user")();
+const cacheData = require("../fixtures/cloudflareCache/data");
+
+const superUser = userData[4];
+
+chai.use(chaiHttp);
+
+describe("Purged Cache Metadata", function () {
+  let jwt;
+
+  after(async function () {
+    await cleanDb();
+    sinon.restore();
+  });
+
+  describe("GET /cache", function () {
+    before(async function () {
+      await cleanDb();
+      const userId = await addUser(userData[0]);
+      jwt = authService.generateAuthToken({ userId });
+    });
+
+    it("Should return last purged cache metadata if no cache logs in last 24 hours found", function (done) {
+      sinon.stub(logsQuery, "fetchLastAddedCacheLog").returns(cacheData.cacheModelData);
+      chai
+        .request(app)
+        .get("/cache")
+        .set("cookie", `${cookieName}=${jwt}`)
+        .end((error, response) => {
+          if (error) {
+            return done(error);
+          }
+
+          expect(response).to.have.status(200);
+          expect(response.body).to.be.an("object");
+          expect(response.body.message).to.be.a("string");
```

Comment:
where are we checking whether the correct message is passed sent or not the server can send anything of type string?

Reply:
Made the improvements

Reply:
Can we keep the exact message? fix it in other places as well

Then we can get it merged bu today EOD